src/FsAutoComplete/LspServers/AdaptiveFSharpLspServer.fs

```diff
@@ -3034,8 +3028,6 @@ module AdaptiveFSharpLspServer =
           None
       | _ -> None
 
-    let _strategy = StreamJsonRpcTracingStrategy(Tracing.fsacActivitySource)
```

Comment:
@TheAngryByrd is this correct? do we need the tracing strategy thing?

Reply:
No there's one defined below, unsure how I got two of them